recipes/DALL-E/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "DALL-E" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/DALL-E-{{ version }}.tar.gz
+    sha256: 55ff0205997e06a1973a559be3fa534ff025e056bd4deff2a88ea4bfec7032a1
+  - url: https://raw.githubusercontent.com/openai/DALL-E/master/LICENSE
+    sha256: 0c6934917db7a46d3b540be58b834baa40824f401aba71f2d6ac7820abf7a27b
+  - url: https://raw.githubusercontent.com/openai/DALL-E/master/requirements.txt
+    sha256: 2329e6582fffb05f1491c8662784ce895bf93ea641336fd64c3393783f906a55
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+  run:
+    - python >=3.6
+    - pillow
+    - blobfile
+    - mypy
+    - numpy
+    - pytest
+    - requests
+    - pytorch
+    - torchvision
+
+test:
+  imports:
+    - dall_e
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://github.com/openai/DALL-E
+  summary: PyTorch package for the discrete VAE used for DALL·E.
+  license: LicenseRef-MIT
```

Comment:
They have a modified MIT license.

Reply:
Maybe it should be LicenseRef-Modified-MIT ?

Reply:
Makes sense!